code/modules/mob/language.dm

```diff
@@ -268,6 +268,17 @@
 	flags = RESTRICTED | WHITELISTED
 	syllables = list("click","clack")
 
+/datum/language/moth
+	name = "Gypsikin"
+	desc = "The noise made vibrating air inside hollow chanbers."
+	speech_verb = "buzzes"
+	ask_verb = "buzzes"
+	exclaim_verb = "buzzes loudly"
+	colour = "moth"
+	key = "m"
```

Comment:
Change this to something else e.g. `mo` as it interferes with medbay's radio channel